### PR TITLE
JS-grpc generation

### PR DESCRIPTION
**Pull request by zoontek**
**work in progress**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,4 +9,4 @@ install:
 .PHONY: test
 test:	build
 	cd examples/dummy && make
-	cd examples/go-kit && make
+	cd examples/js-grpc && make
```
Comment (moul) [inline Makefile]: Keep both tests :)

**Comment by zoontek** [comment]
Yes, it's temporary as go-kit test fails :)

**Comment by moul** [comment]
Oops ok, maybe @quentinperez has a few minutes to give us ? :)

**Comment by zoontek** [comment]
Should do the trick for now. All the necessary funcs for rn / fow generation are here, and will only make sense in the protodefs repo.

Unfortunately no support for imports right now.

**Pull request merged**